lastminute.yml

```diff
@@ -92,6 +92,12 @@
         state: absent
       when: root_password is defined
 
+    - name: Remove icpc2023 user
+      ansible.builtin.user:
+        name: "icpc2023"
+        state: absent
+      when: root_password is defined
+
```

Comment:
I think you can loop over this as you do the same for the `icpc` user under the same condition.

Reply:
just pushed that.